anvil/src/udev.rs

```diff
@@ -247,13 +246,6 @@ pub fn run_udev(log: Logger) {
             state.output_map.borrow_mut().refresh();
         }
     }
-
-    // Cleanup stuff
-    state.window_map.borrow_mut().clear();
-
-    event_loop.handle().remove(session_event_source);
-    event_loop.handle().remove(libinput_event_source);
-    event_loop.handle().remove(udev_event_source);
```

Comment:
As far as I understand, removal of these should happen automatically through RAII and order doesn't matter.

Of course this can be done explicitly, but doing so is likely confusing when it is unnecessary. I haven't noticed any issues, should this code actually be **necessary** please let me know.